config/betterquesting/DefaultQuests/Quests/Tier0StoneAge-AAAAAAAAAAAAAAAAAAAAAQ==/AroundandAroundS-AAAAAAAAAAAAAAAAAAAHPA==.json

```diff
@@ -46,6 +46,7 @@
         "0:10": {
           "id:8": "bartworks:BW_CombinedRotor",
           "Count:3": 1,
+          "tag:10": {},
```

Comment:
what does that do? or is supposed to do?

Reply:
If anything in this PR is a non-text change, it was made automatically by `/bq_admin default save`.